dali/imgcodec/parsers/webp_test.cc

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <string>
+#include <vector>
+#include "dali/imgcodec/image_source.h"
+#include "dali/imgcodec/parsers/webp.h"
+#include "dali/test/dali_test.h"
+#include "dali/test/dali_test_config.h"
+
+namespace dali {
+namespace imgcodec {
+namespace test {
+
+class WebpParserTest : public ::testing::Test {
+ private:
+  WebpParser parser_;
+  const std::string webp_directory_ = testing::dali_extra_path() + "/db/single/webp/";
+
+ public:
+  WebpParserTest() {}
```

Comment:
Is the explicit ctor necessary? Could we get rid of it?

Reply:
No it is not, I can remove it.